uiautomator/__init__.py

```diff
@@ -404,6 +402,31 @@ def install(self):
         for apk in self.__apk_files:
             self.adb.cmd("install", "-r -t", os.path.join(base_dir, apk)).wait()
 
+    def get_forwarded_port(self):
+        for s, lp, rp in self.adb.forward_list():
```

Comment:
s, lp, rp could be named better. Also, I know this module doesn't have the best documentation, but I think a docstring would be good.